chaos-crds/templates/argo-crds.yaml

```diff
@@ -0,0 +1,845 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
do we need to provide an option if user want to install only chaos crd not argo crds?

Reply:
To run chaos infra, both chaos and argo crds are required.

Reply:
I mean if user already have argo crd due to some other tech stack